tests/unit/CIGitLogicTest.sh

```diff
@@ -389,10 +390,10 @@ update_staging_from_main
 tag_staging
 
 # Verify production release list
-assert_prs_merged_between '1.0.0-2' '1.0.1-4' "[ 9, 7, 6, 5, 2 ]"
+assert_prs_merged_between '1.0.0-2' '1.0.1-4' "[ 9, 7, 6, 5, 3, 2 ]"
```

Comment:
All the places the PR 3 has been added is because this PR was CPed to 1.0.1 [here](https://github.com/Expensify/App/blob/5b487e465742fc8b9f244f0204bb286612f41141/tests/unit/CIGitLogicTest.sh#L236)